hermeto/core/package_managers/cargo/main.py

```
@@ -97,7 +106,27 @@ def purl(self) -> PackageURL:
 
     def to_component(self) -> Component:
         """Convert CargoPackage into SBOM component."""
-        return Component(name=self.name, version=self.version, purl=self.purl.to_string())
+        ref_rest = dict(type=PROXY_REF_TYPE, comment=PROXY_COMMENT)
```

Ruff (0.15.21)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)